src/components/Patient/Utils.ts

```diff
@@ -17,3 +23,24 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+
+export const useAddPatientNote = (options: {
+  patientId: string;
+  thread: PatientNotesModel["thread"];
+  consultationId?: string;
+}) => {
+  const queryClient = useQueryClient();
+  const { patientId, thread, consultationId } = options;
+
+  return useMutation({
+    mutationFn: mutate(routes.addPatientNote, {
+      pathParams: { patientId },
+    }),
+    onSuccess: (data) => {
```

Comment:
We can add the success notification back in here. 

Also, might as well add translations for the notifications while you are at it. Other than that, lgtm.

Reply:
do we need a success notification for sending a message in the first place? 😅 
feels kinda weird.

Reply:
🤔  Hmm, yea you are right 😅  Let's skip the success translation then.